doc/source/compatibility.rst

```diff
@@ -4,20 +4,30 @@
 Compatibility
 =============
 
-BioSimSpace has been tested against the following versions of its
-external dependencies:
+BioSimSpace has been tested against the versions of its
+external dependencies listed below. Please let us know if
+you encounter problems by raising an issue on our
+`GitHub <https://github.com/michellab/BioSimSpace/issues>`__
+page. (It may also work with more recent versions and we
+will update these lists as they are validated.)
 
 
 `AMBER <http://ambermd.org>`__
 ==============================
 
-* AmberTools20 (Except CMAP support, e.g. ff19SB.)
+* AmberTools22
+* AmberTools21
+* AmberTools20
 * AmberTools19
 * AmberTools18
 
+Note that AmberTools >= 20 is required for Open Force Field support.
+We currently have support for force fields with CMAP terms, e.g. ff19SB.
+
 `GROMACS <http://www.gromacs.org>`__
 ====================================
 
+* 2021 series
```

Comment:
Do we wish to support 2022 as well?

Reply:
Yes, as stated in the documentation it's just that we've not _fully_ tested against 2022 yet. There is a section stating that this does not mean that it won't work against more recent versions, only that it hasn't been fully battle hardened. We generally update at the end of the year for GROMACS, i.e. when we haven't experienced issues for _all_ releases from that year.